paasta_tools/metrics/metastatus_lib.py

```diff
@@ -39,6 +39,7 @@
 EXPECTED_HEALTHY_FRAMEWORKS = 2
 HIGH_QUEUE_GAUGE = 'org.apache.mesos.chronos.scheduler.jobs.TaskManager.highQueueSize'
 QUEUE_GAUGE = 'org.apache.mesos.chronos.scheduler.jobs.TaskManager.queueSize'
+MAINTENANCE_RESOURCE = 'maintenance'
```

Comment:
It would be nice to be able to import this value from somewhere and share that with `paasta_tools/mesos_maintenance.py` (which currently just hard codes `maintenance`